multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/query/providers/SqlFileQueryProvider.java

```diff
@@ -156,16 +173,41 @@ public SqlQuery<List<FileEntry>> getListFilesQuery(String space, String namespac
         };
     }
 
-    public SqlQuery<List<FileEntry>> getListFilesQuery(String space, String namespace, String fileName) {
+    public SqlQuery<List<FileEntry>> getListFilesBySpaceAndOperationId(String space, String operationId) {
         return (Connection connection) -> {
             PreparedStatement statement = null;
             ResultSet resultSet = null;
             try {
                 List<FileEntry> files = new ArrayList<>();
-                statement = connection.prepareStatement(getQuery(SELECT_FILES_BY_NAMESPACE_SPACE_AND_NAME));
-                statement.setString(1, namespace);
-                statement.setString(2, space);
-                statement.setString(3, fileName);
+                statement = connection.prepareStatement(getQuery(SELECT_FILES_BY_SPACE_AND_OPERATION_ID));
+                statement.setString(1, space);
+                // TODO: The namespace is currently used as operation ID, and will be set for backwards compatibility for 1 tact
```

Comment:
it might affect the query a bit, otherwise it won't be compatible and will lead to missing logs
not needed for the other query, this only applies to process logs

Reply:
actually these methods with the OR clause are only used when getting the logs (cf dmol...)

Reply:
tested with stress tests, no impact on performance